flink-core/src/test/java/org/apache/flink/api/common/state/StateTtlConfigTest.java

```diff
@@ -71,4 +72,27 @@ public void testStateTtlConfigBuildWithCleanupInBackground() {
         assertThat(incrementalCleanupStrategy.runCleanupForEveryRecord(), is(false));
         assertThat(rocksdbCleanupStrategy.getQueryTimeAfterNumEntries(), is(1000L));
     }
+
+    @Test
+    public void testStateTtlConfigBuildWithWithCustomIncrementalCleanupStrategy() {
+        int zeroCleanupSize = 0;
```

Comment:
We can loop an array of `illegalCleanUpSize`s to reduce code duplication.

Reply:
Hi Myasuka, done.